internal/cli/common.go

```diff
@@ -0,0 +1,64 @@
+package cli
+
+import (
+	"errors"
+
+	"github.com/coreos/airlock/internal/config"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// airlockCmd is the top-level cobra command for `airlock`
+	airlockCmd = &cobra.Command{
+		Use:               "airlock",
+		Short:             "Update/reboot manager, with distributed locking based on etcd3",
+		PersistentPreRunE: commonSetup,
+	}
+
+	configPath  string
+	runSettings *config.Settings
+)
+
+// Init initializes the CLI environment for airlock
+func Init() (*cobra.Command, error) {
+	airlockCmd.PersistentFlags().StringVarP(&configPath, "config", "c", "/etc/airlock/config.toml", "path to configuration file")
+
+	airlockCmd.AddCommand(cmdServe)
+
+	logrus.SetLevel(logrus.DebugLevel)
```

Comment:
Should we default to a higher log level and add a `verbose` flag now to enable debug logging?

Reply:
Added a counting `-v` flag. Its zero default now maps to `WarnLevel`.